nanofi/examples/CMakeLists.txt

```diff
@@ -19,57 +19,35 @@
 
 cmake_minimum_required(VERSION 3.16)
 
-include_directories(/include)
-
-include(CheckCXXCompilerFlag)
-if (WIN32)
-  if ((MSVC_VERSION GREATER "1900") OR (MSVC_VERSION EQUAL "1900"))
-	    CHECK_CXX_COMPILER_FLAG("/std:c++14" _cpp_latest_flag_supported)
-	    if (_cpp_latest_flag_supported)
-	        add_compile_options("/std:c++14")
-	    endif()
-	endif()
-else()
-CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
-CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
-if(COMPILER_SUPPORTS_CXX11)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-elseif(COMPILER_SUPPORTS_CXX0X)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
-else()
- message(STATUS "The compiler ${CMAKE_CXX_COMPILER} has no C++11 support. Please use a different C++ compiler.")
-endif()
-
-endif()
+include(CppVersion)
+set_cpp_version()
 
 if (NOT WIN32)
+    add_executable(generate_flow generate_flow.c)
 
-add_executable(generate_flow generate_flow.c)
-
-add_executable(terminate_handler terminate_handler.c)
-
-target_wholearchive_library(generate_flow minifi-http-curl)
-target_link_libraries(generate_flow nanofi Threads::Threads)
+    add_executable(terminate_handler terminate_handler.c)
 
-target_link_libraries(terminate_handler nanofi Threads::Threads)
+    target_wholearchive_library(generate_flow minifi-http-curl)
+    target_link_libraries(generate_flow nanofi Threads::Threads)
 
-#add_executable(hash_file hash_file.c)
+    target_link_libraries(terminate_handler nanofi Threads::Threads)
 
-#arget_link_libraries(hash_file nanofi Threads::Threads )
+    #add_executable(hash_file hash_file.c)
 
-add_executable(transmit_flow transmit_flow.c)
+    #arget_link_libraries(hash_file nanofi Threads::Threads )
```

Comment:
(old code) Can we add a comment why this is commented out?

Reply:
I don't know the reason, just fixed the indentation here after changing the compiler flags above.

Reply:
It looks like the whole executable is commented out for whatever reason.

Reply:
We could write a Jira ticket for it to investigate and fix the issue later.

Reply:
I don't really see the point of further pursuing this as long as nanofi is barely maintained.
This is where hash_file was introduced: #499 
And this is where it was removed: #514 [MINIFICPP-767 Fix issue with hash](https://issues.apache.org/jira/browse/MINIFICPP-767)

Reply:
Yeah, that's okay then, thanks!